composer.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "prooph/pdo-event-sourcing-pack",
+    "description": "A pack to install and pre-configure Prooph PDO event sourcing components in a Symfony Flex application",
+    "type": "symfony-pack",
+    "license": "MIT",
+    "keywords": ["event sourcing"],
+    "homepage": "http://getprooph.org/",
+    "authors": [
+        {
+            "name": "Gildas Quéméner",
+            "email": "gildas.quemener@gmail.com"
+        }
+    ],
+    "require": {
+        "prooph/service-bus-symfony-bundle": "^0.6.0",
+        "prooph/event-store-symfony-bundle": "^0.4.0",
+        "prooph/event-sourcing": "^5.3",
+        "prooph/pdo-event-store": "^1.7"
```

Comment:
`prooph/event-store-bus-bridge` and `react/promise` are missing.

Reply:
Indeed, I haven't had time to dive into their configuration yet (https://github.com/gquemener/minimal-symfony-prooph-event-store/pull/1)

Reply:
Could you point me out why `react-promise` is required please @UFOMelkor?

Reply:
See https://github.com/prooph/service-bus-symfony-bundle/issues/63 (third point)